package.json

```diff
@@ -131,6 +131,7 @@
     "lerna": "^3.13.4",
     "lint-staged": "^8.1.6",
     "lit-element": "^2.1.0",
+    "lit-html": "^1.1.1",
```

Comment:
Why do you install lit-html. OK, for `ifDefined `

Reply:
Yeah, exactly